pkg/controller/v1beta1/inferenceservice/components/transformer.go

```diff
@@ -152,45 +153,79 @@ func (p *Transformer) Reconcile(ctx context.Context, isvc *v1beta1.InferenceServ
 
 	// Here we allow switch between knative and vanilla deployment
 	if p.deploymentMode == constants.RawDeployment {
-		r, err := raw.NewRawKubeReconciler(ctx, p.client, p.clientset, p.scheme, objectMeta, metav1.ObjectMeta{},
-			&isvc.Spec.Transformer.ComponentExtensionSpec, &podSpec, nil)
-		if err != nil {
-			return ctrl.Result{}, errors.Wrapf(err, "fails to create NewRawKubeReconciler for transformer")
-		}
-		// set Deployment Controller
-		for _, deployment := range r.Deployment.DeploymentList {
-			if err := controllerutil.SetControllerReference(isvc, deployment, p.scheme); err != nil {
-				return ctrl.Result{}, errors.Wrapf(err, "fails to set deployment owner reference for transformer")
-			}
+		if err := p.reconcileTransformerRawDeployment(ctx, isvc, &objectMeta, &podSpec); err != nil {
```

Comment:
nit: From what I can tell the previous code was _mostly_ just shifted to the newly created function, but this file doesn't really re-use the same lines in multiple places from what I could see. So I'm kind of unsure as to the intent behind separating this out into a function.